### ISSUE TITLE
Search Results: Tag links in list layout provide no positive benefit

### ISSUE THREAD

**Issue by sympmarc** [microsoft-search/pnp-modern-search#1729]
### Check existing issues before logging new ones
I don't *think* there's an existing issue for this, but I could have missed it.

**Version used**
Ex: 4.5.3 (But I think all 4.x)

**Describe the bug**
When using the **List** layout in the **Search Results** (having added `Auto` before `owstaxidmetadataalltagsinfo` in **Layout slots** and turned on localization) we get links for all **Tags** used. Here's a screenshot:

![image](https://user-images.githubusercontent.com/1295627/149984999-03649a91-46a2-4bf6-afe4-ca0dca9d15ef.png)

**Expected behavior**
The links add text like `#521%20-%20Milling%20and%20Flotation,%20D40%20-%20Mill%20Feed%20System:'521%20-%20Milling%20and%20Flotation'` to the URL, but that has no effect.

It would seem there are short-term and long-term solutions:
- Short-term - Disable the links in the item footer. Seeing the Tags is useful, but since the links don't accomplish anything, they are misleading and confusing.
- Long-term - Make the links work. My assumption would be they should allow the user to click on the specific tag and then the Search Results will be filtered by that value.

The operative code is in the template:
``` handlebars
                                    <div class="template--listItem--tags example-themePrimary">
                                        {{#if (slot item @root.slots.Tags)}}
                                        <pnp-icon data-name="Tag" aria-hidden="true" data-theme-variant="{{JSONstringify @root.theme}}"></pnp-icon>
                                        {{#each (split (slot item @root.slots.Tags) ",") as |tag| }}
                                        <a href="#{{slot item @root.slots.Tags}}:'{{trim tag}}'">{{tag}}</a>&nbsp;{{/each}} {{/if}}
                                    </div>
```

Changing line 5 from:
``` handlebars
<a href="#{{slot item @root.slots.Tags}}:'{{trim tag}}'">{{tag}}</a>
```
to
``` handlebars
<div>{{tag}}</div>
```
would seem to be the short-term fix. I can do a PR for this, but I wanted to make sure I'm barking up the right tree. I'd really like to see the long-term fix, but I'm not sure how to go about it.

**Desktop (please complete the following information):**
 - Browser [e.g. chrome, safari] - Any

**Comment by wobba** [comment]
This came up in the PnP AMA I believe and @FranckyC proposed as you also do to remove the "click" target on the tags. If you want to PR a change to the OOB templates with tags that would be appreciated.

**Comment by sympmarc** [comment]
I was the one who brought it up in the AMA. :) I wanted to discuss it here so we could figure out what the best approach was before I did something. Just removing the link sweeps it under the rug, but it's a short-term fix.

**Comment by wobba** [comment]
@sympmarc There is no quick fix, as if you want the click to work it's dependent on having a search box in the mix, and also what parameter you have used to trigger this. I'd rather remove it, and people can re-add click behavior matching their setup as needed.

**Comment by sympmarc** [comment]
Okey-dokey. So replacing the links with just displaying the info in those three files should be the "fix"?

**Comment by FranckyC** [comment]
> Okey-dokey. So replacing the links with just displaying the info in those three files should be the "fix"?

Yes basically. As mentioned in the AMA session, this was more an example of what you could do with it, like grab the URL fragment and do a subsequent query using tokens. You can just remove the link in template, so no confusion anymore. I'm fine with this.

**Comment by FranckyC** [comment]
@sympmarc finally fixed with #1787. No link anymore.

**Comment by sympmarc** [comment]
Sorry - I was thinking about this today. I meant to do it, but you beat me.
